sql/updates/0.10.1--0.11.0-dev.sql

```diff
@@ -0,0 +1,4 @@
+DROP FUNCTION IF EXISTS drop_chunks(INTERVAL, NAME, NAME, BOOLEAN);
+DROP FUNCTION IF EXISTS drop_chunks(ANYELEMENT, NAME, NAME, BOOLEAN);
+DROP FUNCTION IF EXISTS _timescaledb_internal.drop_chunks_impl(BIGINT, NAME, NAME, BOOLEAN, BOOLEAN)
+DROP FUNCTION IF EXISTS _timescaledb_internal.dimension_get_time(INTEGER);
```

Comment:
depending on which release this function ends up in, we should remember to put this in relevant update file.